src/commands/validate.ts

```diff
@@ -21,26 +22,34 @@ export default class Validate extends Command {
   async run() {
     const { args, flags } = await this.parse(Validate); //NOSONAR
     const filePath = args['spec-file'];
-
     const watchMode = flags['watch'];
 
-    const specFile = await load(filePath);
-    if (watchMode) {
-      specWatcher({spec: specFile, handler: this, handlerName: 'validate'});
-    }
     try {
+      const specFile = await load(filePath);
+
+      if (watchMode) {
+        specWatcher({ spec: specFile, handler: this, handlerName: 'validate' });
+      }
+
       if (specFile.getFilePath()) {
         await parser.parse(specFile.text());
         this.log(`File ${specFile.getFilePath()} successfully validated!`);
       } else if (specFile.getFileURL()) {
         await parser.parse(specFile.text());
         this.log(`URL ${specFile.getFileURL()} successfully validated`);
       }
-    } catch (error) {
-      throw new ValidationError({
-        type: 'parser-error',
-        err: error
-      });
+    } catch (e) {
+      if (e instanceof SpecificationFileNotFound) {
+        this.error(new ValidationError({
```

Comment:
I think you should not trigger a `ValidationError` but instead a [SpecificationFileNotFoundError](https://github.com/asyncapi/cli/blob/master/src/errors/specification-file.ts#L9). This way, we'll get the right message when the file doesn't exist.